gkgen/minlength_test.go

```diff
@@ -0,0 +1,107 @@
+package gkgen
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMinLengthGenerator(t *testing.T) {
+	type minLengthStruct struct {
+		Bool      bool
+		BoolPtr   *bool
+		String    string
+		StringPtr *string
+	}
+
+	t.Run("RequireParam", func(t *testing.T) {
+		t.Parallel()
+		assert := require.New(t)
+		validator := NewMinLengthValidator()
+		s := minLengthStruct{String: "foo"}
+		st := reflect.TypeOf(s)
+		field, found := st.FieldByName("String")
+		assert.True(found)
+
+		_, err := validator.Generate(field, []string{})
+		assert.Error(err)
+	})
+
+	t.Run("RequireParamIsANumber", func(t *testing.T) {
```

Comment:
ok nw i think i get it now testing []string{"wat"} and not minLengthStruct{int: 3}